packages/hyperion-autologging/src/ALSurface.ts

```diff
@@ -355,7 +355,7 @@ export function init(options: InitOptions): ALSurfaceHOC {
           triggerFlowlet: callFlowlet.data.triggerFlowlet
         });
       }
-    }, [domAttributeName, domAttributeValue, callFlowlet, triggerFlowlet, nodeRef]);
+    }, [domAttributeName, domAttributeValue]);
```

Comment:
How about the noderRef? Shouldn't we keep that otherwise we miss new nodes added to the dom?